src/components/backdropscreensaver/plugin.js

```diff
@@ -1,19 +1,15 @@
-define(["connectionManager"], function (connectionManager) {
-
-    return function () {
-
-        var self = this;
-
-        self.name = "Backdrop ScreenSaver";
-        self.type = "screensaver";
-        self.id = "backdropscreensaver";
-        self.supportsAnonymous = false;
-
-        var currentSlideshow;
```

Comment:
Where did this variable go?

Reply:
It's a class property now and thus can't be declared